src/styles/components/_carousel.scss

```diff
@@ -1,5 +1,39 @@
 $mc-carousel-centered-padding: 100px !default;
 
+.slick-dots {
```

Comment:
Passing `dotsClass` to slick slider would allow us to set a custom class here.

Reply:
If you want to change the name we can, otherwise this is fine too.  If we don't change the name, maybe we would want to wrap this in something like `.mc-carousel` so that we don't affect other instances that might have dots?